src/Core/Package.vala

```diff
@@ -82,6 +83,26 @@ public class AppCenterCore.Package : Object {
         REMOVING
     }
 
+    public enum PermissionsFlags {
```

Comment:
```suggestion
    [Flags]
    public enum PermissionsFlags {
```

If you do this, you can remove all of the `1 << x` parts. Probably don't need `LAST` in here either.

Reply:
Oh that's great! Much more maintainable